### ISSUE TITLE
CSS Overflow Navigation Controls

### ISSUE THREAD

**Issue by flackr** [WebKit/standards-positions#447]
### Title of the proposal

CSS Overflow 5: Scroll navigation controls

### URL to the spec

https://drafts.csswg.org/css-overflow-5/#scroll-navigation

### URL to the spec's repository

https://github.com/w3c/csswg-drafts/

### Explainer URL

https://github.com/w3c/csswg-drafts/blob/main/css-overflow-5/carousel-explainer.md

### TAG Design Review URL

https://github.com/w3ctag/design-reviews/issues/1037

### Mozilla standards-positions issue URL

https://github.com/mozilla/standards-positions/issues/1161

### Description

There are a [variety of problems being solved by carousels](https://css.oddbird.net/overflow/explainer/), which we believe could be provided by a small set of CSS features. Developers could then combine these CSS features to create the various designs. CSS-only component libraries could be built to further simplify this process.

**Comment by danielsakhapov** [comment]
FYI, we've update scroll-marker-group property to support modes to better comply with WCAG patterns - https://chromestatus.com/feature/5109685301673984?gate=6191471012216832

**Comment by fantasai** [comment]
Some issues that seem pretty fundamental:
- https://github.com/w3c/csswg-drafts/issues/12240
- https://github.com/w3c/csswg-drafts/issues/12269
